src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -149,9 +149,12 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
         GetMappingsRequest getMappingsRequest = buildGetMappingRequest(indexName);
         client.admin().indices().getMappings(getMappingsRequest, ActionListener.<GetMappingsResponse>wrap(getMappingsResponse -> {
             Map<String, MappingMetadata> mappings = getMappingsResponse.getMappings();
+            if (mappings.size() == 0) {
+                throw new IllegalArgumentException("No matching index with index name: " + indexName);
```

Comment:
Seems we should use`No matching mapping with ...`, is this index from user's input?